src/ui/pages/Identifiers/Identifiers.tsx

```diff
@@ -149,24 +149,8 @@ const Identifiers = () => {
     if (timeB === null) return -1;
     return timeA - timeB;
   });
-  const handlePendingClick = async (identifier: IdentifierShortDetails) => {
-    // @TODO - sdisalvo: This is a temporary fix Patrick initially added to the CardStack
-    // and I moved it here since PendingIdentifiers are never going to show up in the stack.
-    /**The below code only return false if the identifier is a multisig and it is not ready */
-    const checkMultisigComplete =
-      await Agent.agent.multiSigs.checkMultisigComplete(identifier.id);
-    if (!checkMultisigComplete) {
-      return;
-    } else {
-      const updatedIdentifiers = identifiersData.map((item) => {
-        if (item.id === identifier.id && item.isPending) {
-          return { ...item, isPending: false };
-        }
-        return item;
-      });
-      dispatch(setIdentifiersCache(updatedIdentifiers));
-      setToggleClick(!toggleClick);
-    }
+  const handlePendingClick = async () => {
+    setToggleClick(!toggleClick);
```

Comment:
From what I can tell, we can remove the `toggleClick` state variable, and the `onCardClick`. @sdisalvo-crd can you confirm?

Also @sdisalvo-crd looking at the `useEffect` above which sorts `identifiersData` into the separate functions. imo this should be a single for loop that appends it to a temp array for each one - rather than calling `.filter` several times which is iterating over the same data again and again. If you are OK with this, I would ask Patrick to do this

Reply:
Sure @iFergal please feel free to go ahead with that change.

Reply:
@iFergal @sdisalvo-crd I have updated this file.

Reply:
Thanks @Sotatek-HocNguyena , I think we can also remove the onClick event and the useState associated with it since we don't need to click on it anymore, is that right @iFergal ?